pkg/plugins/external/helpers.go

```diff
@@ -102,8 +105,52 @@ func makePluginRequest(req external.PluginRequest, path string) (*external.Plugi
 	return &res, nil
 }
 
+// getUniverseMap is a helper function that is used to read the current directory to build
```

Comment:
Would be possible to add a unit test for this implementation?

Reply:
yes sure, I'll add a unit test for this, thank you for the feedback!

Reply:
@camilamacedo86 I've added a unit test in [fd9cfa](https://github.com/kubernetes-sigs/kubebuilder/pull/3223/commits/fd9cfa14aecf13de66bf8000af21435c2e461282), let me know if it needs to be adjusted or to add more test cases. 
thank you!